frontend/src/services/index.ts

```diff
@@ -1 +1,12 @@
-export * from './http/http.service';
+import { Http } from './http/http.service';
+import { UserApi } from './user-api/user-api.service';
+import { ENV } from 'common/enums';
+
+const http = new Http();
+
+const userApi = new UserApi({
+  http,
+  apiPrefix: (ENV.API_PATH as string)
```

Comment:
```suggestion
  apiPrefix: ENV.API_PATH
```
Merge with dev to avoid it